about.php

```diff
@@ -68,7 +68,7 @@
 		<h3>Tools We Used</h3>
 		<p>Like any other company, the founders of StarBulls relied on various resources and tools that helped
 			them
-			get the site up and running! We would like to share these resources/tools with the
+			get the site up and running! We want to share these resources and tools with the
 			world.
```

Comment:
why is this spread over so many lines like this?

Reply:
this has been formatted